### ISSUE TITLE
Не работает shakerMode, если используем ComponentActivity

### ISSUE THREAD

**Issue by EmogurovAnton** [RedMadRobot/debug-panel-android#8]
Если MainActivity наследуется от ComponentActivity, то shakerMode работает только в самой панели после перехода через уведомление. Если наследуемся от FragmentActivity, то проблем нет

**Comment by Zestxx** [comment]
Да, такая проблема действительно существует и о ней известно. Проблема связана с тем что в текущей версии панели для навигации используется FragmentManager получаемый через `supportFragmentManager` FragmentActivity.  По хорошему чтобы все нормально работало нужно перевести панель на Compose. Частично это уже реализовано и надеюсь в ближайшее время я добью полную миграцию на Compose и эта проблема исчезнет.
